### PR TITLE
【PTen】Refactor the directory of fill kernels

### PR DESCRIPTION
**Pull request by zyfncg**
### PR types

Others

### PR changes

Others

### Describe

PTen中fill系列kernel目录重构

### CODE REVIEW

**Review by MingMingShangTian — DISMISSED** [review]

**Review by YuanRisheng — DISMISSED** [review]

File: paddle/pten/kernels/eigen/fill_kernel.cc [hunk 1]
```diff
@@ -0,0 +1,44 @@
+/* Copyright (c) 2021 PaddlePaddle Authors. All Rights Reserved.
```
Comment (chenwhql) [inline paddle/pten/kernels/eigen/fill_kernel.cc]: format error

File: paddle/pten/kernels/eigen/fill.h [hunk 2]
```diff
@@ -0,0 +1,73 @@
+/* Copyright (c) 2021 PaddlePaddle Authors. All Rights Reserved.
```
Comment (chenwhql) [inline paddle/pten/kernels/eigen/fill.h]: fill -> full

File: paddle/pten/kernels/eigen/fill_kernel.cc (hunk 1 above)
Comment (zyfncg) [reply paddle/pten/kernels/eigen/fill_kernel.cc]: Done

File: paddle/pten/kernels/eigen/fill.h (hunk 2 above)
Comment (zyfncg) [reply paddle/pten/kernels/eigen/fill.h]: Done

**Review by chenwhql — APPROVED** [review]

**Pull request merged**